tests/syntax_tests/data/ppx/react/expected/typeConstraint.res.txt

```diff
@@ -2,11 +2,11 @@
 
 module V4C = {
   @res.jsxComponentProps
-  type props = {}
+  type props<'a, 'b> = {a: 'a, b: 'b}
```

Comment:
@mununki are these changes good or bad?

Reply:
Add test to analysis (so it's compiled and uses rescript-react as a dependency) that these compile.
Tested that the same examples did not compile on master, so this is the intended transformation that was somehow lost along the way sometime.


https://github.com/rescript-lang/rescript/pull/7200/commits/de232620996edf7e304f1c9ed6555be3ded5031f